test/valgrind/valgrindTestSuite.sh

```diff
@@ -575,13 +614,18 @@ then
         mv /tmp/accumulator_$LISTENER_PORT       $vtest.accumulator_$LISTENER_PORT
         mv /tmp/accumulator_$LISTENER2_PORT      $vtest.accumulator_$LISTENER2_PORT
 
-        failText=$(failedTest "$vtest.valgrind.out" $vtest 0 "valgrind" $xTestNo)
+        failText=$(leakFound "$vtest.valgrind.out" $vtest 0 "valgrind" $xTestNo)
```

Comment:
Actually, in the vtest section, both functions are called as well.
I didn't pay very much attention to that part as I would like to remove it completely.
`failText` is just the text that is written after each case.

I haven't tried the vtest. It just may work perfectly.
I vote for removing it instead of making it "perfect".